psydac/core/bsplines.py

```diff
@@ -728,6 +742,7 @@ def basis_ders_on_quad_grid(knots, degree, quad_grid, nders, normalization, out=
             [[0. , 0.03125, 0.125, 0.28125]]]])
     """
     ne, nq = quad_grid.shape
+    knots = np.asarray(knots)
```

Comment:
As above

Reply:
Done in https://github.com/pyccel/psydac/commit/54fa1fb4c2afba79c77392308a193bad59161fee